src/coord/src/session/vars.rs

```diff
@@ -237,7 +237,17 @@ impl Vars {
         } else if name == DATABASE.name {
             self.database.set(value)
         } else if name == DATE_STYLE.name {
-            bail!("parameter {} is read only", DATE_STYLE.name);
+            for value in value.split(',') {
```

Comment:
not sure if this matters but i think this permits the empty string?

Reply:
not actually true whoops :)

Reply:
No worries!